omics_orientation/omics_orientation.md

```diff
@@ -0,0 +1,195 @@
+
+### Quiz: Omics topics
+
+Which of the following omics topics involves studying gene expression by quantifying or sequencing RNA?
+
+[( )] Genomics
+[(X)] Transcriptomics
+[( )] Proteomics
+[( )] Metabolomics
+***
+<div class = "answer">
+Transcriptomics is the study of the RNA transcripts in a system, which is considered one measure of gene expression under certain conditions. Genomics is the study of the genome, which is DNA; proteomics deals with proteins, and metabolomics studies the small molecules resulting from cell metabolism.
+</div>
+***
+
+## Caveats and Challenges
+
+While omics approaches can be powerful, this doesn't mean that an omics approach is always the "best", nor that omics studies are guaranteed to provide every answer to every question! There are several caveats to be aware of when planning or reading about an omics study:
+
+* Heterogeneity of populations: According to Hasin et al., "[i]nsights gained from omics approaches are mostly comparative" (2017), meaning that omics studies involve comparing groups to each other (such as "disease vs. healthy" or "treatment vs control", for example). Inherent heterogeneity in any of these populations makes it difficult to distinguish important differences, or "signal", from unrelated variation, also called "noise." Because of the amount of data being analyzed in omics studies, this problem can be amplified compared to traditional approaches.
+
+* Confounding variables: Omics allows researchers to ask complex questions with many variables, but that doesn't mean studies can include every possible variable. Variables that researchers don't know about or don't measure can still be contributing to the effect in question. Any conclusions drawn from the results of an omics study (or any scientific study) should account for this possibility.
+
+* Reproducibility: Due to the amount and complexity of the data in omics studies, the analysis process can require many steps, and there will be differences in workflows between studies. This means that results sometimes can't be reproduced from one study to another.
+
+### Quiz: Caveats and Challenges
+
+True or False: A well-designed omics study will allow a researcher to pinpoint the cause of a specific disease in an individual.
+
+[( )] TRUE
+[(X)] FALSE
+***
+<div class = "answer">
+While a well-designed omics study might offer some insight into factors that could contribute to the disease in question (which in turn could reveal promising treatment targets or more information about disease prognosis), study populations are simply too heterogeneous and there are too many possible variables for a study to lead to a definite conclusion about a specific individual.
```

Comment:
Maybe spell this out a little more clearly in the caveats section. I'm thinking a short paragraph like "While omics studies can _leverage large data sets to find patterns and correlations_ the don't _provide much insight about individual participants_" or something along  those lines so that this quiz question feels more directly connected to the content that precedes it.

Reply:
Good call, I like that!